config/settings/base.py

```diff
@@ -237,3 +240,24 @@
 CLIENT_VERIFY_CERTIFICATES = strtobool(
     os.getenv("ROSETTA_CLIENT_VERIFY_CERTIFICATES", "True")
 )
+
+
+SENTRY_DEBUG_URL_ENABLED = False
+if SENTRY_DSN := os.getenv("SENTRY_DSN", ""):
+    sentry_sdk.init(
+        dsn=SENTRY_DSN,
+        environment=os.getenv("SENTRY_ENVIRONMENT", ""),
```

Comment:
Make this a more generic `ENVIRONMENT_NAME` where the default is `production` (e.g. https://github.com/search?q=repo%3Anationalarchives%2Fds-frontend%20ENVIRONMENT_NAME&type=code)